test/run-tests.py

```diff
@@ -534,7 +547,13 @@ def RunTest(info, options, variables, verbose_level=0):
   cwd = REPO_ROOT_DIR
   gen_input_path = info.CreateInputFile()
   rel_gen_input_path = os.path.relpath(gen_input_path, cwd)
-  variables['out_dir'] = os.path.dirname(rel_gen_input_path)
+
+  out_dir = os.path.splitext(rel_gen_input_path)[0] + "_out"
```

Comment:
So this would convert a test named `test/parse/expr/br.txt` to an output directory named `out/test/parse/expr/br_out`, right?

Can we just use the name without "_out" instead?